SConstruct

```diff
@@ -128,6 +130,7 @@ command_line_options = {
     'forcetopology':    ['0','1'],
     'debug':            ['0','1'],
     'noadaptivesync':   ['0','1'],
+    'cryptoengine':     [''],                               # Can be any board-specific name

```

Comment:
CRYPTO_ENGINE translates to the name of the board-specific implementation, i.e. the structure gathering different function pointers. See for instance the declaration of cc2538_crypto_engine struct in bsp/boards/OpenMote-CC2538/cc2538_crypto_engine.c. 

We could for instance make a convention that all board-specific implementations are called "board_crypto_engine" and then wait for this in SCons. This will allow only one CRYPTO_ENGINE implementation per board though.